client/src/components/shared/Info/Finances.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+import React, {Fragment} from 'react'
+import {compose, withState, withHandlers} from 'recompose'
+import {Badge, Button} from 'reactstrap'
+import ChevronUp from 'react-icons/lib/fa/chevron-up'
+import ChevronDown from 'react-icons/lib/fa/chevron-down'
+
+import type {StateUpdater} from '../../../types/commonTypes'
+import type {EnhancedCompanyFinancial} from '../../../services/utilities'
+import {icoUrl, ShowNumberCurrency} from '../../../services/utilities'
+import Item from './Item'
+import Trend from './Trend'
+import './Finances.css'
+import './InfoButton.css'
+
+type FinancesProps = {|
+  data: EnhancedCompanyFinancial[],
+  ico: string,
+  toggledOn: boolean,
+  toggle: () => void,
+|}
+
+type StateProps = {
+  toggledOn: boolean,
+  toggle: StateUpdater<boolean>,
+}
+
+const FinancesItem = ({data, ico}: {data: EnhancedCompanyFinancial, ico: string}) => {  
+  const {employees, profit, profitTrend, revenue, revenueTrend, year} = data
```

Comment:
You could do `{data: {employees, profit, profitTrend, revenue, revenueTrend, year}, ico}` on L#28 and remove braces and `return` :slightly_smiling_face: Also, I prefer defining `FinancesItemProps` type, it looks less clumsy than defining the type inline.

Reply:
What's your opinion on defining multiple components in one file? I prefer spliting them up, but I know that we do it like this a lot in this project.

Reply:
Yes, I absolutely agree with you. I also prefer one component in one file. I've done it this way because I always try to imitate and keep the code style of the project I'm working on ... I will split it.

Just a note. I also prefer to have exactly one scss/css file for each component and for the main style use the same name as component has, i.e. for "Item" it will be ".Item". We used to do it in this way on Dermoi but here it is done in other way... Never mind.